uportal-war/src/main/webapp/WEB-INF/jsp/Search/searchLauncher.jsp

```diff
@@ -35,10 +35,12 @@
         <c:set var="searchLabel"><spring:message code="search"/></c:set>
         <form class="form-search" role="form" method="post" action="${searchLaunchUrl}" id="${n}webSearchForm">
           <div class="input-group">
-            <input id="${n}webSearchInput"  class="searchInput input-large search-query form-control" value="" name="query" type="text" placeholder="Enter search terms"/>
+            <input id="${n}webSearchInput" class="searchInput input-large search-query form-control" value="" name="query" type="search" placeholder="Enter search terms" title="Search Terms"/>
```

Comment:
Both title and placeholder have been updated as localized strings, with English :gb: and French :fr: translations. :sparkles: